src/lib/edra/headless/editor.css

```
@@ -490,25 +495,51 @@
+ * font-size, so mixed sizes become staggered segments.
+ * Paint a mid-line on the text itself (inline + box-decoration-break) so the
+ * stroke only covers glyphs, not the full row width. font-size on <p> makes
+ * the fragment height follow the largest size; unmarked runs are pinned.
+ */
+.tiptap ul[data-type='taskList'] li[data-checked='true'] > div > p {
+	display: inline;
+	font-size: var(--edra-task-font-size, 1em);
+	text-decoration: none;
+	box-decoration-break: clone;
+	-webkit-box-decoration-break: clone;
+	background-image: linear-gradient(currentColor, currentColor);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)